File: 2535.difference-between-element-sum-and-digit-sum-of-an-array.java
/*
 * @lc app=leetcode id=2535 lang=java
 *
 * [2535] Difference Between Element Sum and Digit Sum of an Array
 */

// @lc code=start
class Solution {
    public int differenceOfSum(int[] nums) {
        int elementSum = 0;
        int digitSum = 0;
        for (int i : nums) {
            elementSum += i;
            while (i > 0) {
                digitSum += i % 10;
                i /= 10;
            }
        }
        return Math.abs(elementSum - digitSum);
    }
}
// @lc code=end

File: 2535.difference-between-element-sum-and-digit-sum-of-an-array.py
#
# @lc app=leetcode id=2535 lang=python
#
# [2535] Difference Between Element Sum and Digit Sum of an Array
#

# @lc code=start
class Solution(object):
    def differenceOfSum(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        es = 0
        ds = 0
        for i in nums:
            es += i
            while i > 0:
                ds += i % 10
                i //= 10
        return abs(es - ds)
# @lc code=end

